src/components/compound/DlCharts/charts/DlConfusionMatrix/DlConfusionMatrix.vue

```diff
@@ -277,7 +283,10 @@ export default defineComponent({
         const { variables } = useThemeVariables()
         const currentBrushState = ref<{ min: number; max: number }>({
             min: 0,
-            max: props.matrix.length
+            max:
```

Comment:
id just use Math.min in this case as to add readability

Reply:
True, I'll change it